build/source/engine/soilLiqFlx.f90

```diff
@@ -1235,19 +1218,21 @@ USE soil_utils_module,only:hydCondMP_liq       ! compute hydraulic conductivity
     ! (compute derivative in hydraulic conductivity w.r.t. temperature)
     dHydCond_dTemp = hydCond_noIce*dIceImpede_dT + dHydCondMicro_dTemp*iceImpedeFac
     ! (test derivative)
-    !xConst = LH_fus/(gravity*Tfreeze)                            ! m K-1 (NOTE: J = kg m2 s-2)
-    !vTheta = scalarVolFracIceTrial + scalarVolFracLiqTrial
-    !volLiq = volFracLiq(xConst*(scalarTempTrial+dx - Tfreeze),vGn_alpha,theta_res,theta_sat,vGn_n,vGn_m)
-    !volIce = vTheta - volLiq
-    !effSat = (volLiq - theta_res) / (theta_sat - volIce - theta_res)
-    !psiLiq = matricHead(effSat,vGn_alpha,0._dp,1._dp,vGn_n,vGn_m)  ! use effective saturation, so theta_res=0 and theta_sat=1
-    !hydCon = hydCond_psi(psiLiq,scalarSatHydCond,vGn_alpha,vGn_n,vGn_m)
-    !call iceImpede(volIce,f_impede,iceImpedeFac,dIceImpede_dLiq)
-    !hydIce = hydCon*iceImpedeFac
-    !print*, 'test derivative: ', (psiLiq - scalarMatricHeadTrial)/dx, dPsiLiq_dTemp
-    !print*, 'test derivative: ', (hydCon - hydCond_noIce)/dx, dHydCondMicro_dTemp
-    !print*, 'test derivative: ', (hydIce - scalarHydCond)/dx, dHydCond_dTemp
-    !pause
+    if(testDeriv)then
+     xConst = LH_fus/(gravity*Tfreeze)                            ! m K-1 (NOTE: J = kg m2 s-2)
```

Comment:
Ensured consistency here. It is a larger job to ensure consistency throughout the code, and I'm going to save this glorious task for later.